### DISCUSSION TITLE
How to Disable enter fullscreen mode automatically based on the device's width in vidstack react player

### DISCUSSION THREAD

**Discussion by sachinkumar121** [vidstack/player#996] [Q&A]
I am using this [player](https://www.npmjs.com/package/@vidstack/react) in one of my React projects. I am showing the video player in a modal. So if the screen width of the device is below 1024px then I don't want to apply the auto-fullscreen mode.

I read the [official documentation](https://www.vidstack.io/docs/player) but didn't get the official way to disable entering fullscreen mode automatically for a specific device's width.

**Comment by mihar-22** [comment]
Set playsinline on the player. New section on [`playsinline` in the installation docs](https://www.vidstack.io/docs/player/getting-started/installation/react?provider=video&styling=default-layout#9-set-playsinline-optional).


There was a small bug with it too that got resolved today, make sure you're on 1.5.3 🙂
